livedata/src/main/java/io/getstream/chat/android/livedata/usecase/UseCaseHelper.kt

```diff
@@ -66,6 +66,24 @@ data class UseCaseHelper(var chatDomainImpl: ChatDomainImpl) {
      * The API call to create the message is retried using the retry policy
      */
     var sendMessage: SendMessage = SendMessageImpl(chatDomainImpl)
+    /**
+     * Cancel an emphemeral message. This message is immediately removed from local storage.
+     * The API call to delete the message is retried using the retry policy
+     */
+    var cancelMessage: CancelMessage = CancelMessage(chatDomainImpl)
```

Comment:
Those properties can be inmutable

Reply:
making all the props of this class immutable